google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java

```diff
@@ -80,6 +80,12 @@
   /** Whether {@link #getContent()} should return raw input stream. */
   private final boolean returnRawInputStream;
 
+  /** Content encoding for GZip */
+  private static final String CONTENT_ENCODING_GZIP = "gzip";
+
+  /** Content encoding for GZip HTTP 1.1 */
+  private static final String CONTENT_ENCODING_XGZIP = "x-gzip";
```

Comment:
Neat. Didn't know about this one. For the comment though, this is a legacy value, not the recommended value in HTTP 1.1.

Reply:
Please see latest commit.